Course_Part_04/src/Assignment16.java

```diff
@@ -1,5 +1,6 @@
 public class Assignment16 {
```

Comment:
Ook deze uitwerking ziet er goed uit!

Hoe zou je ook getal 1 kunnen meenemen in de isPrime ?